test/SparqlAntlrParserTest.cpp

```diff
@@ -2052,6 +2051,24 @@ auto notExistsMatcher(Matcher<const ParsedQuery&> pattern) {
   return builtInCallTestHelpers::matchNaryWithChildrenMatchers(
       &makeUnaryNegateExpression, existsMatcher(pattern));
 }
+
+// _____________________________________________________________________________
+std::vector<const SparqlExpression*> getExistsExpressions(
+    const SparqlFilter& filter) {
+  std::vector<const SparqlExpression*> result;
+  filter.expression_.getPimpl()->getExistsExpressions(result);
+  return result;
+}
```

Comment:
This is the second time you need this functionality
(also in the actual parser), consider 
1. making it a member function of `SparqlFilter` or 
2. at least add the overload for getExistsExpressions that returns a vector directly, s.t. you only need one line here (the sparql parser + its tests are already long enugh).

Reply:
The "problem" is that we still need two versions: 1 const and 1 non-const, but yeah I'll change it